Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
And does that test scenario work with non ISO-8859-1 characters (try Russian, Greek and Chinese) after that PR was applied?